pkg/types/options.go

```diff
@@ -170,6 +170,8 @@ type Options struct {
 	DisableRedirects bool
 	// PathClimb enables path expansion (auto crawl discovered paths)
 	PathClimb bool
+	// DisableUniqueFilter disables duplicate content filtering
+	DisableUniqueFilter bool
```

Comment:
We can use more [positive](https://refactoring.com/catalog/removeDoubleNegative.html) naming here -minor